test/e2e/run_restart_test.go

```diff
@@ -0,0 +1,39 @@
+package integration
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman run restart containers", func() {
+	var (
+		tempdir    string
+		err        error
+		podmanTest PodmanTest
+	)
+
+	BeforeEach(func() {
+		tempdir, err = CreateTempDirInTempDir()
+		if err != nil {
+			os.Exit(1)
+		}
+		podmanTest = PodmanCreate(tempdir)
+		podmanTest.RestoreAllArtifacts()
+	})
+
+	AfterEach(func() {
+		podmanTest.Cleanup()
+	})
+
+	It("Podman start after successful run", func() {
+		session := podmanTest.Podman([]string{"run", ALPINE, "ls"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+
+		session2 := podmanTest.Podman([]string{"start", "--attach", "--latest"})
+		session2.WaitWithDefaultTimeout()
+		Expect(session2.ExitCode()).To(Equal(0))
+	})
+})
```

Comment:
Is there any value in making a test where we had a non-graceful exit to the container and check that start works?  For example, some sort of kill on the pid in the container ... and then start?

Reply:
Yeah, that's a good idea, I'll add a test